app/Main.hs

```diff
@@ -164,6 +165,15 @@ installOrUpdate set pkgName PackageInfo{ repo, version } = do
     cloneShallow repo version pkgDir
   pure pkgDir
 
+getReverseDeps  :: PackageSet -> PackageName -> IO [(PackageName, PackageInfo)]
+getReverseDeps db dep =
+    foldMap go $ Map.toList db
+  where
+    go pair@(_, PackageInfo {dependencies}) =
+      case List.find (== dep) dependencies of
```

Comment:
This will only give immediate reverse dependencies, but we need transitive reverse dependencies.